.travis/install_test.sh

```diff
@@ -4,7 +4,7 @@ set -ev
 mkdir --parents "${HOME}/bin"
 
 # PHPUnit install
-wget "https://phar.phpunit.de/phpunit-5.7.phar" --output-document="${HOME}/bin/phpunit"
+wget "https://phar.phpunit.de/phpunit-7.phar" --output-document="${HOME}/bin/phpunit"
```

Comment:
Isn't this now handled by dev-kit?

Reply:
It is, but that doesn't matter because I am going to handle the conflict in #568